app/connectors/RegistrationConnector.scala

```diff
@@ -32,4 +33,9 @@ class RegistrationConnector @Inject()(config: Configuration, httpClient: HttpCli
   def getRegistration()(implicit hc: HeaderCarrier): Future[Option[Registration]] =
     httpClient.GET[Option[Registration]](s"$baseUrl/registration")
 
+  def getRegistration(vrn: Vrn): Future[Option[Registration]] = {
+    implicit val emptyHc: HeaderCarrier = HeaderCarrier()
```

Comment:
Why do we need empty headercarrier here? And can we get it implicitly as per method above

Reply:
The HeaderCarrier in the other method comes from a request. Since this a background task there isn't really a header carrier. And I wouldn't want there to be one as otherwise it'd use the VRN from that, rather than the one from the return